kapitan/initialiser.py

```diff
@@ -20,27 +20,151 @@
 import os
 import sys
 import shutil
+import yaml
+
 from distutils.dir_util import copy_tree
+from kapitan.utils import list_all_paths
 
 logger = logging.getLogger(__name__)
 
-def initialise_skeleton(directory):
-    """ Initialises a directory with a recommended skeleton structure
-    Args:
-        directory (string): path which to initialise, directory is assumed to exist
-    """
-
-    current_pwd = os.path.dirname(__file__)
-    templates_directory = os.path.join(current_pwd, 'inputs', 'templates')
-
-    copy_tree(templates_directory, directory)
-
-    logger.info("Populated {} with:".format(directory))
-    for dirName, subdirList, fileList in os.walk(directory):
-        logger.info('{}'.format(dirName))
-        for fname in fileList:
-            logger.info('\t {}'.format(fname))
-        # Remove the first entry in the list of sub-directories
-        # if there are any sub-directories present
-        if len(subdirList) > 0:
-            del subdirList[0]
+class initialise_skeleton(object):
```

Comment:
Camel case for classes

Reply:
Yup, will do it . Earlier it was written like this. So, i used it.

Reply:
Yes it was a function, not a class :)